modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/DeployService.java

```diff
@@ -268,6 +267,7 @@ private List<DeployResourceEntity> getDeployResourceEntityList(
      * Get deployment and fill deployTask for destroy service task.
      *
      * @param deployTask the task of deploy managed service name.
+     * @param userId     the id of user who deployed the service.
      */
     public Deployment getDestroyHandler(DeployTask deployTask, String userId) {
```

Comment:
@baixinsui Do we still need the "userId" parameter here?

Reply:
Removed